tests/cases/typecheck/property_access/numeric_access_on_struct.stderr

```diff
@@ -1,6 +1,6 @@
 error[0015]: the field `2` is not present within `Frobulate(data = 23_i32, other = 'c')`
- --> $DIR/numeric_access_on_struct.hash:8:6
+ --> $DIR/numeric_access_on_struct.hash:8:1
 7 |   
 8 |   t := Frobulate(data = 23);
-  |        ^^^^^^^^^^^^^^^^^^^^ `Frobulate(data = 23_i32, other = 'c')` does not contain the property `2`
```

Comment:
Is this not a regression?

Reply:
It possibly makes more sense since those errors are due to variable access.. I can change it back to the way it was too.